.github/actions/verify-badge/action.yaml

```diff
@@ -0,0 +1,34 @@
+name: Verify-Badge
+description: Perform verification to assist with UDS badge certification
+
+inputs:
+  chartPath:
+    description: Relative path to the directory with the uds-config chart
+    default: chart/
+  groupName:
+    description: The name of the niche/group that the package fits into. E.g. package,swf,lfai
+    default: package
+  commonZarf:
+    description: Whether or not there is a common zarf.yaml file
+    default: "true"
+
+runs:
+  using: composite
+  steps:
+    - name: Setup Go environment
+      uses: actions/setup-go@v5.0.2
+      with:
+        go-version: 1.22
+        check-latest: true
+
+    - name: Helm tool installer
+      uses: Azure/setup-helm@v4.2.0
```

Comment:
these should be pinned, but renovate should do it for you iirc once there is an update to the versions 🤔

Reply:
Yeah I assume renovate will take care of this

Reply:
would prefer to default to do this if we notice it since technically if that repo was compromised our CI could be as well - in this repo it wouldn't give much more than the CG read creds but would still prefer to update prior to merge / adoption.